src/artifact.rs

```diff
@@ -6,14 +6,14 @@ pub enum Artifact {
     Example(String),
 }
 
-impl AsRef<Path> for Artifact {
-    fn as_ref(&self) -> &Path {
-        Path::new(match self {
-            Self::Root(_) => "",
-            Self::Example(_) => "examples",
-        })
-    }
-}
+// impl AsRef<Path> for Artifact {
+//     fn as_ref(&self) -> &Path {
+//         Path::new(match self {
+//             Self::Root(_) => "",
+//             Self::Example(_) => "examples",
+//         })
+//     }
+// }
```

Comment:
keep or remove

Reply:
Is it used anywhere? I still have to link this against `cargo-apk` but it seems rather wonky to me, only returning an empty path or `"examples"` instead of somehow returning the target path? What was the intended use here?

Reply:
It is in fact being used as a build directory in at least 3 places in `cargo-apk`... I think we should make a more explicit `fn build_dir(&self) -> &'static Path {}` for this one.